core/src/main/scala/org/broadinstitute/dsde/rawls/fastpass/FastPassServiceImpl.scala

```diff
@@ -509,26 +520,28 @@ class FastPassServiceImpl(protected val ctx: RawlsRequestContext,
   }
 
   protected def writeGrantsToDb(workspaceId: String,
-                                userAndPet: UserAndPetEmails,
+                                userAndPets: UserAndPetEmails,
                                 samUserSubjectId: WorkbenchUserId,
                                 gcpResourceType: IamResourceType,
                                 resourceName: String,
                                 organizationRoles: Set[String],
                                 expiration: OffsetDateTime
   )(implicit dataAccess: DataAccess): ReadWriteAction[Unit] = {
     val rolesToWrite =
-      Seq((userAndPet.userEmail, userAndPet.userType), (userAndPet.petEmail, IamMemberTypes.ServiceAccount)).flatMap(
-        tuple => organizationRoles.map(r => (tuple._1, tuple._2, r))
-      )
-    DBIO.seq(rolesToWrite.map { tuple =>
+      (userAndPets.petEmails.map((_, IamMemberTypes.ServiceAccount)).toSeq :+ (userAndPets.userEmail,
```

Comment:
Am I missing a difference between this and the other places you're using `.toSeq` or was this just missed?